src/CollAction/Views/Projects/Index.cshtml

```diff
@@ -47,33 +53,39 @@
 @foreach (var item in Model) {
         <tr>
             <td>
-                @Html.DisplayFor(content => item.Name)
+                @Html.DisplayFor(content => item.Project.Name)
+            </td>
+            <td>
+                @Html.DisplayFor(content => item.Project.Description)
+            </td>
+            <td>
+                @Html.DisplayFor(content => item.Project.Goal)
             </td>
             <td>
-                @Html.DisplayFor(content => item.Description)
+                @Html.DisplayFor(content => item.Project.Category.Name)
             </td>
             <td>
-                @Html.DisplayFor(content => item.Goal)
+                @Html.DisplayFor(content => item.Project.Location.Name)
             </td>
             <td>
-                @Html.DisplayFor(content => item.Category)
+                @Html.DisplayFor(content => item.Project.Target)
             </td>
             <td>
-                @Html.DisplayFor(content => item.Location)
+                @Html.DisplayFor(content => item.Project.Start)
             </td>
             <td>
-                @Html.DisplayFor(content => item.Target)
+                @Html.DisplayFor(content => item.Project.End)
             </td>
             <td>
-                @Html.DisplayFor(content => item.Start)
+                @Html.DisplayFor(content => item.Project.StatusDescription)
             </td>
             <td>
-                @Html.DisplayFor(content => item.End)
+                @Html.DisplayFor(content => item.Progress)%
             </td>
             <td>
-                <a asp-action="Edit" asp-route-id="@item.Id">Edit</a> |
-                <a asp-action="Details" asp-route-id="@item.Id">Details</a> |
-                <a asp-action="Delete" asp-route-id="@item.Id">Delete</a>
+                <a asp-action="Edit" asp-route-id="@item.Project.Id">@Localizer["Edit"]</a> |
```

Comment:
This is still stuff left over from the auto-generated project views that asp.net can generate for you. I don't think it makes a lot of sense to show edit and delete buttons to everyone in all the project lists.